packages/cadl-ranch-specs/http/special-headers/request-id/main.tsp

```diff
@@ -0,0 +1,27 @@
+import "@typespec/http";
+import "@azure-tools/cadl-ranch-expect";
+import "@azure-tools/typespec-client-generator-core";
+
+using TypeSpec.Http;
+using Azure.ClientGenerator.Core;
+
+@doc("Non-standard azure request id header configurations.")
+@supportedBy("dpg")
+@scenarioService("/special-headers/request-id")
+namespace SpecialHeaders.RequestId;
+
+@scenario
+@scenarioDoc("""
+Non-standard request id header. SDK should not genreate `clientRequestId` paramerter but use policy to auto-set the header.
+Expected header parameters:
+- x-ms-client-request-id=<any uuid string>
+""")
+@doc("""
+Non-standard request id header.
+""")
+@get
+@route("/non-standard")
+op nonStandard(
```

Comment:
I would maybe not say `nonStandard` as the scenario name. EIther change to something like `valid` or better just move `@scenario` and `@scenarioDoc` on the namespace so the whole file is the scenario.